src/App.vue

```diff
@@ -1,13 +1,18 @@
 <template>
   <div id="app">
-    <img src="./assets/logo.png">
+    <app-header></app-header>
     <router-view/>
   </div>
 </template>
 
 <script>
+import header from './components/Header.vue'
```

Comment:
Ima bolji nacin za ovo. :) 
1. Uvek se trudi da ti komponente budu iz 2 reci, da bi izbegao konflikte sa npr. web komponentama. 
Tako da, slobodno nazovi komponentu `AppHeader`
2. `import AppHeader from './components/AppHeader.vue'`
I onda dole samo kazes:
```
components: {
  AppHeader
}
```
On ce u template-u prepoznati da su to 2 reci, po velikim slovima, sa hyphen-om izmedju. :)